Sources/AWSLambdaRuntime/LambdaHandlers.swift

```diff
@@ -175,11 +176,15 @@ public struct ClosureHandler<Event: Decodable, Output>: LambdaHandler {
 
 extension LambdaRuntime {
     /// Initialize an instance with a ``StreamingLambdaHandler`` in the form of a closure.
-    /// - Parameter body: The handler in the form of a closure.
+    /// - Parameter
+    ///   - logger: The logger to use for the runtime. Defaults to a logger with label "LambdaRuntime".
+    ///   - body: The handler in the form of a closure.
     public convenience init(
+        logger: Logger = Logger(label: "LambdaRuntime"),
```

Comment:
`LambdaRuntime.init(handler:eventLoop:logger)` forces the log level to be `info` if there is no `LOG_LEVEL` environment variable. Perhaps it should just use the supplied Logger logLevel if the environment variable is not there

Reply:
excellent idea. I wonder why we didn't do that earlier